rdrf/rdrf/models/definition/models.py

```diff
@@ -1157,6 +1157,9 @@ class EmailNotificationManager(models.Manager):
     def subscribable(self):
         return self.filter(subscribable=True)
 
+    def is_registry_subscribable(self, registries):
```

Comment:
Not necessary, but this function name feels a bit off to me.
Can you think of anything better e.g. `any_subscribable`, `has_any_subscribable` , `has_subscribable_registry`?
Also happy for it to be left as-is.